internal/runners/show/show.go

```diff
@@ -158,24 +162,71 @@ func (s *Show) Run(params Params) error {
 		return locale.WrapError(err, "err_show_secrets", "Could not get secret information")
 	}
 
-	data := outputData{
-		ProjectURL:   projectURL,
-		Namespace:    fmt.Sprintf("%s/%s", owner, projectName),
+	rd := RuntimeDetails{
+		NameSpace:    fmt.Sprintf("%s/%s", owner, projectName),
 		Name:         projectName,
 		Organization: owner,
 		Visibility:   visibilityData(owner, projectName, remoteProject),
-		Commit:       commit,
-		Languages:    languages,
-		Platforms:    platforms,
-		Secrets:      secrets,
-		Events:       events,
-		Scripts:      scripts,
+		LastCommit:   commit,
+	}
+
+	outputData := outputData{
+		ProjectURL:     projectURL,
+		RuntimeDetails: rd,
+		Languages:      languages,
+		Platforms:      platforms,
+		Secrets:        secrets,
+		Events:         events,
+		Scripts:        scripts,
 	}
 
+	data := output.NewFormatter(outputData).WithFormat(output.PlainFormatName, output.NewStackedOutput([]interface{}{
+		"Here are the details of your runtime environment.",
+		"",
+		struct {
+			*RuntimeDetails `opts:"verticalTable"`
+		}{&rd},
+		"",
+		output.Heading(locale.Tl("state_show_events_header", "Events")),
+		events,
+		"",
+		output.Heading(locale.Tl("state_show_scripts_header", "Scripts")),
+		formatScripts(scripts),
+		"",
+		output.Heading(locale.Tl("state_show_platforms_header", "Platforms")),
+		platforms,
+		"",
+		output.Heading(locale.Tl("state_show_languages_header", "Languages")),
+		languages,
+	}))
```

Comment:
This is neat but totally goes against convention. Please stick to our existing patterns.

Definitely reaching the point where we need to take another look at how we handle output, but this isn't the story for that.